leaf-ui/rappid-extensions/LinkInspector.js

```diff
@@ -170,11 +170,16 @@ var LinkInspector = Backbone.View.extend({
             
         } else {
             //Verify if it is possible to remove the NB tag from source and target
-            if (!this.hasNBLink(source, this.cell) && this.hasNBTag(source)){
+            if (!this.hasNBLink(source, this.cell) && !this.hasNBTag(source)){
+                
                 source.attr(".funcvalue/text", "");
```

Comment:
This is out of scope for your changes - but I would recommend switching the quotations in lines 175 and 181 to single quotes to match lines 176 and 182